features/step_definitions/installation.rb

```diff
@@ -11,11 +11,9 @@
 
 # peform offline install using ccz file pushed from repository
 Then (/^I install the ccz app at "([^\"]*)"$/) do |path|
-  step("I rotate to portrait")
```

Comment:
@ctsims I wasn't sure of value it's serving since phones are already in portrait when we install the apps! Can include if you think it's worth.

Reply:
I think sometimes when we get to this state we were in a broken 'pre-state' that was left in landscape, and since the tests inside of install relied on rotation, this was here to set things up, but if it's causing issues I'm not super opposed to removing it.

Reply:
Added it back.